cloudhub-2/modules/ROOT/pages/ps-config-fw-rules.adoc

```diff
@@ -82,3 +84,176 @@ image::fw-rule-delete.png[Delete icon on the Firewall rules tab]
+     "runtimeVersion": "4.4.0:20220523-1",
+     "updateStrategy": "rolling",
+     "disableAmLogForwarding": false,
+     "generateDefaultPublicUrl": true
+   },
+   "replicas": 1
+ },
+ "application": {
+   "ref": {
+     "groupId": "66310c16-bce5-43c4-b978-5945ed2f99c5",
+     "artifactId": "app-with-40-schedulers",
+     "version": "1.0.5",
+     "packaging": "jar"
+   },
+   "assets": [],
+   "desiredState": "STARTED",
+   "configuration": {
+     "mule.agent.application.properties.service": {
+       "applicationName": "logs-app-uc01",
+       "properties": {},
+       "secureProperties": {}
+     },
+     "mule.agent.logging.service": {
+       "artifactName": "logs-app-uc01",
+       "scopeLoggingConfigurations": []
+     }
+   },
+   "integrations": {
+     "services": {
+       "objectStoreV2": {
+         "enabled": false
+       }
+     }
+   },
+   "vCores": "0.05"
+ }
+}
+----
+
+Deployments that include unavailable ports fail. Successful deployments occupy the newly mapped ports, and those ports become unavailable. If the app is deleted, the ports become available again.
+
+=== Update an App Using the Application Manager API
+
+In the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/amc-application-manager/minor/4.0/console/method/%231158/[Application Manager API], use the `tcp` section under `deploymentSettings` to update TCP port and redeploy your app. 
+
+The following example shows request to update an existing app and its body with a new TCP mapping:
+
+[source,json]
+----
+PATCH
+https://anypoint.mulesoft.com/amc/adam/api/organizations/{organizationId}/environments/{environmentId}/deployments/{deploymentId}
+
+{
+    "id": "id",
+    "target": {
+      "provider": "MC",
+      "targetId": "873a9879-cca7-4211-b90e-826e98123929",
+      "deploymentSettings": {
+        "tcp": {
+          "inbound": {
+            "ports": [
+              {
+                "portNumber": 30507,
+                "applicationPortNumber": 9007
+              },
+              {
+                "portNumber": 30508,
+                "applicationPortNumber": 9008
+              },
+              {
+                "portNumber": 30509,
+                "applicationPortNumber": 9009
+              }
+            ]
+          }
+        }
+      }
+    }
+  }
+----
+
+Updates that include unavailable ports fail. Successful deployments occupy the newly mapped ports, and those ports become unavailable. The ports that you replaced with new mappings become available.
+
+=== Access Your TCP App
+
+You can access the app using VPN or a transit gateway and within the private space. Since the apps run on worker nodes, they cannot be accessed directly over the internet. The URL to access a TCP app has this structure: `{application-url}.tcp.{environment}.cloudhub.io:{port}`
+
+For example, the URL for the app deployed in the previous examples would be `logs-app-uc01.tcp.cloudhub.io:30500`.
```

Comment:
the example URL is missing {environment} - is that optional?

Reply:
@rsaxena05 is the env relevant for EU customers? Or can we not mention it?